packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Axes/index.tsx

```diff
@@ -0,0 +1,267 @@
+                            setMax={(newValue) => setXMaxValue(0, newValue)}
+                        />
+                    )}
+                    <Group spacing="xs">
+                        <Group spacing="xs">
+                            <Config.SubLabel>Sort</Config.SubLabel>
+                            <SegmentedControl
+                                defaultValue={
+                                    dirtyEchartsConfig?.xAxis?.[0]?.inverse
+                                        ? 'descending'
+                                        : 'ascending'
+                                }
+                                data={[
+                                    {
+                                        value: 'ascending',
+                                        label: (
+                                            <MantineIcon
+                                                icon={IconSortAscending}
+                                            />
+                                        ),
+                                    },
+                                    {
+                                        value: 'descending',
+                                        label: (
+                                            <MantineIcon
+                                                icon={IconSortDescending}
+                                            />
+                                        ),
+                                    },
+                                ]}
+                                onChange={(value) => {
+                                    setInverseX(value === 'descending');
+                                }}
+                            />
+                        </Group>
+                        {!dirtyLayout?.flipAxes && (
+                            <Group noWrap spacing="xs" align="baseline">
+                                <Config.SubLabel>Rotation</Config.SubLabel>
+                                <NumberInput
+                                    type="number"
+                                    defaultValue={
+                                        dirtyEchartsConfig?.xAxis?.[0].rotate ||
+                                        0
+                                    }
+                                    min={0}
+                                    max={90}
+                                    step={15}
+                                    maw={54}
+                                    rightSection="°"
+                                    onChange={(value) => {
+                                        setXAxisLabelRotation(Number(value));
+                                    }}
+                                />
+                            </Group>
+                        )}
+                    </Group>
+                </Config.Group>
+            </Config>
+
+            <Config>
+                <Config.Group>
+                    <Config.Label>{`${
```

Comment:
These are like, group labels, right? 

Oh I see, you have Label and SubLabel. I think I'd prefer GroupLabel for the groups and just Label for the sub things. I think it might be clearer how to use them. 

Bu this is *very minor*, feel free to ignore.

Reply:
yes they are! I like that change yes. 
Lemme change everywhere!
> I think it might be clearer how to use them.

Makes sense!

Reply:
Ok, one question before I make the change: 
There's already a `LabelGroup`, which is a Group for the Label and it can have another child element like the `AddButton`. 
If I have `GroupLabel`, then there's two v similar components. What would you suggest doing in this case?

Reply:
> Are you saying there's already a Config.LabelGroup?

yup, there's one already! 

So, with this in mind: 
- `Config.Group`
- `Config.Heading` - was `Label` before
- `Config.GroupHeading` - was `GroupLabel` before
- `Config.Label` - was `SubLabel` before

Reply:
What do you think? ^